themes/default/content/docs/guides/crosswalk/aws/api-gateway.md

```diff
@@ -47,242 +50,261 @@ We will see later how to assign a custom domain, SSL certificate, and/or elimina
 
 There are multiple ways to define APIs using Pulumi Crosswalk for AWS:
 
-* [Lambda Function Event Handler Route](#defining-a-lambda-function-event-handler-route)
-* [Static Route Served by S3](#defining-a-static-route-served-by-s3)
-* [Integration Route](#defining-an-integration-route)
-* [OpenAPI Specification for an Entire Endpoint](#defining-an-openapi-specification-for-an-entire-endpoint)
-* [OpenAPI Specification for a Single Route](#defining-an-openapi-specification-for-a-single-route)
+- [Lambda Function Event Handler Route](#defining-a-lambda-function-event-handler-route)
+- [Static Route Served by S3](#defining-a-static-route-served-by-s3)
+- [Integration Route](#defining-an-integration-route)
+- [OpenAPI Specification for an Entire Endpoint](#defining-an-openapi-specification-for-an-entire-endpoint)
+- [OpenAPI Specification for a Single Route](#defining-an-openapi-specification-for-a-single-route)
 
 Multiple endpoints on the same API Gateway can be defined using a combination of these techniques.
 
-### Defining a Lambda Function Event Handler Route
+### Lambda Request Handling
```

Comment:
This title change feels less clear to me.  Any reason not to keep the previous title?

Aside - the links in the list above still use the names of the anchors from the older names.

Reply:
I've been trying to make the title a little shorter so they don't get cut off in the side naviagation:

![image](https://user-images.githubusercontent.com/331676/143436914-baf7bcb4-7e31-4da7-85b0-ebbdcb20bec8.png)

Will just drop the prefix here instead - "Lambda Function Event Handler Route" - and check all the cross-links too 👍